openehr-rm/src/main/java/com/nedap/archie/rminfo/ArchieRMInfoLookup.java

```diff
@@ -291,6 +288,15 @@ public void processCreatedObject(Object createdObject, CObject constraint) {
             Locatable locatable = (Locatable) createdObject;
             locatable.setArchetypeNodeId(constraint.getNodeId());
             locatable.setNameAsString(constraint.getMeaning());
+            if(constraint != null && constraint instanceof CArchetypeRoot) {
```

Comment:
```suggestion
            if(constraint instanceof CArchetypeRoot) {
```

`constraint` is always non-null at this point. A NullPointerException whould have been thrown to lines earlier if it was null.

Reply:
will fix!